src/components/screens/Consul/Debates/DebateDetail.js

```diff
@@ -0,0 +1,177 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { ScrollView } from 'react-native';
+import { useForm } from 'react-hook-form';
+import { useMutation } from 'react-apollo';
+
+import { DefaultKeyboardAvoidingView } from '../../../DefaultKeyboardAvoidingView';
+import { SafeAreaViewFlex } from '../../../SafeAreaViewFlex';
+import { Title, TitleContainer, TitleShadow } from '../../../Title';
+import { Wrapper, WrapperVertical } from '../../../Wrapper';
+import { HtmlView } from '../../../HtmlView';
+import { Button } from '../../../Button';
+import {
+  ConsulCommentList,
+  ConsulTagList,
+  ConsulVotingComponent,
+  ConsulPublicAuthorComponent,
+  Input,
+  ConsulStartNewButton
+} from '../../../Consul';
+import { consts, device, texts } from '../../../../config';
+import { useOpenWebScreen } from '../../../../hooks';
+import { ConsulClient } from '../../../../ConsulClient';
+import { ADD_COMMENT_TO_DEBATE } from '../../../../queries/Consul';
+import { QUERY_TYPES } from '../../../../queries';
+
+const text = texts.consul;
+const a11yText = consts.a11yLabel;
+
+/* eslint-disable complexity */
+/* NOTE: we need to check a lot for presence, so this is that complex */
+export const DebateDetail = ({ listData, onRefresh, route, navigation }) => {
+  const [loading, setLoading] = useState();
+
+  const {
+    cachedVotesDown,
+    cachedVotesUp,
+    cachedVotesTotal,
+    comments,
+    commentsCount,
+    description,
+    id,
+    publicAuthor,
+    publicCreatedAt,
+    tags,
+    title
+  } = listData.debate;
+
+  const openWebScreen = useOpenWebScreen(
+    route.params?.title ?? '',
+    undefined,
+    route.params?.rootRouteName
+  );
+
+  // React Hook Form
+  const { control, handleSubmit, reset } = useForm();
+
+  // GraphQL
```

Comment:
i think this comment is not very helpful, as `useMutation` is imported from `react-apollo`

Reply:
I usually add these comments to navigate through the code.